### PR TITLE
Code Challanges

### PR DESCRIPTION
**Pull request by RaghadAbdulhadi1**

### CODE REVIEW

File: codeChallenges/challenge-1/index.test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  * example usage:
+  * const arrOfNum = [1, 2, 2, 4, 5, 6, 6];
+  * console.log(getUniqueValues(arrOfNum)); // [1, 2, 4, 5, 6]
+  */
+ const arrOfNum = [1, 2, 2, 4, 5, 6, 6];
+ const getUniqueValues = (arrOfNum) => {
+   // TO DO
+   var uniqueNums = [];
+   for(let i = 0; i < arrOfNum.length; i++){
+    if(arrOfNum[i+1] != arrOfNum[i]){
```
Comment (jadabuawwad) [inline codeChallenges/challenge-1/index.test.js]: Here you are checking tow values throw itteration, if they are equaled or on not,
what about [1,2,5,2] ? here if the duplicated values are not follow each other you code will not cover this case.

**Review by jadabuawwad — COMMENTED** [review]
Challange 1 review :

Postive points :
- You are following the branch name convention and commits convention 

Negative Points :
- You got a solution that is not cover all cases 

Final Mark is : 4/5

**Review by jadabuawwad — APPROVED** [review]
Challange 2 review :

Postive points :
- You are following the recursive solution which is the optimal one Great! 

Extra note with no side effect: here we used ES5 function declaration, also we can use the ES6 arrow function, so could you tell me the difference between them ?
  
Final Mark is : 5/5

File: codeChallenges/challenge-1/index.test.js (hunk 1 above)
Comment (RaghadAbdulhadi1) [reply codeChallenges/challenge-1/index.test.js]: resolve at: [refrence](https://github.com/RaghadAbdulhadi1/Training-Raghad/pull/1/commits/d433e02deb8c28c50cbcd84db4fcd28d292eee02)

**Comment by RaghadAbdulhadi1** [comment]
Arrow Functions:
- Fewer lines of code
- No duplicate named parameter
- Can not be invoked with new keyword
- The value of this remains the same inside the arrow function

**Pull request merged**